### DISCUSSION TITLE
Checking on Raspberry Pi 5 and PlayerOne

### DISCUSSION THREAD

**Discussion by barrettepp** [aaronwmorris/indi-allsky#1102] [Q&A] [answered]
I am just in the planning stages of setting up an allsky camera.   My original idea was to your TJ allsky program but I am getting a great deal on a player one Neptune C camera.   So that got me looking into indi-allsky.  I tried to ask this in the facebook group for allsky but none of the post were being accepted.   
My questions is can I run a raspberry pi 5 and the Neptune -C camera.   I just want to make sure it will work before I order the camera.  Thank you

**Comment by aaronwmorris** [comment]
indi-allsky has been tested on Raspberry Pi 5 and Debian bookworm.  All of the issues in that regard have been resolved.  There have been no reported issues with INDI based cameras.

I think you should be good to go.

**Reply by barrettepp** [reply]
Thank you for the reply.

**Comment by czimm74** [comment, marked as answer]
Yes.  I have a Neptune C II running on Pi5 with Bookworm.  Handles all quite well.  Your only time consuming part will be dialing in the camera settings, I am still tweaking.

**Reply by barrettepp** [reply]
Sounds great.   I am still waiting for all my supplies to come in.  Trying to decide how I want to do a case.    It will all come together.

**Reply by czimm74** [reply]
I have separate enclosures for the camera and Pi.  The pi I did put in a case with an active cooler, the case mainly to keep things safe The computer an extra storage are in an enclosure inside my garage, more a personal design choice.. and easier to access the computer if need be.
